src/contrib/persistence/Akka.Persistence.Sql.Common/Journal/BatchingSqlJournal.cs

```diff
@@ -528,6 +535,11 @@ public RequestChunk(int chunkId, IJournalRequest[] requests)
         /// </summary>
         protected readonly bool CanPublish;
 
+        /// <summary>
+        /// The timestamp provider that will be used for the timestamp column when writing messages to the database.
+        /// </summary>
+        protected ITimestampProvider TimestampProvider { get; }
```

Comment:
Need to add API approval for this https://getakka.net/community/public-api-changes.html

Reply:
I took care of this

Reply:
Thanks